.gitignore

```diff
@@ -70,22 +70,23 @@ Pods/
 # For more information about the recommended setup visit:
 # https://docs.fastlane.tools/best-practices/source-control/#source-control
 
-fastlane/README.md
-fastlane/report.xml
-fastlane/Preview.html
-fastlane/screenshots/**/*.png
-fastlane/test_output
-fastlane/xcov_output
+iosApp/fastlane/README.md
+iosApp/fastlane/report.xml
+iosApp/fastlane/Preview.html
+iosApp/fastlane/screenshots/**/*.png
+iosApp/fastlane/test_output
+iosApp/fastlane/xcov_output
 
 # Sourcery
 *.generated.swift
 
 # Derived Data folder
-DerivedData
+iosApp/DerivedData
 
 # Tuist
 .tuist-generated
 
 # JetBrains IDEs
 
 .idea
+GoogleService-*.plist
```

Comment:
Isn't this file is specific for iOS and we should placed it in `iosApp/`, right?

Reply:
Modified the gitignore file. 🙏